scripts/st2client-startup.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
 
+counter=0
+backoff=10
+
 # st2client startup and registration
-while true; do
+while [ "$counter" -lt 5 ]; do
   ACTIONS=$(st2 action list)
   if [ "$?" -ne 0 ]; then
-    echo "unable to reach downstream, will try again"
-    sleep 5
+    counter=$((counter+1))
+    echo "unable to reach downstream, will try again in $backoff seconds..."
+    backoff=$(awk -v backoff="$backoff" 'BEGIN{ printf "%.f", backoff * 1.5 }')
```

Comment:
This should be simpler

```suggestion
    backoff=$(echo $backoff*1.5|bc)
```

Reply:
I tried using bc at first, but it doesn't work here because its not packaged in the image.

Reply:
Makes sense, thanks for more context